zuul/zuul-server.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Zuul Server Service
+After=syslog.target network.target
+ 
+[Service]
+Type=simple
+User=zuul
+Group=zuul
+#EnvironmentFile=-/etc/sysconfig/zuul
```

Comment:
Is this required #EnvironmentFile=-/etc/sysconfig/zuul ?

Reply:
No. Removed.